src/Serilog/Formatting/Display/MessageTemplateTextFormatter.cs

```diff
@@ -91,6 +91,15 @@ public void Format(LogEvent logEvent, TextWriter output)
 
                     token.Render(overridden, output, _formatProvider);
                 }
+                else if (pt.IsLevelProperty)
```

Comment:
I see that the alignment still needs to be passed through; might still need the conditional but rather than `pi.IsLevelProperty` it could be `pt.Value is LogEventLevelValue`? (using the as/null check pattern if possible)